docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -310,9 +310,7 @@ There are also some tasks which are generated internally and implicitly at vario
 Connection plugins
 ------------------
 
-Connection plugins allow Ansible to connect to the target hosts so it can execute tasks on them. Ansible ships with many connection plugins, but only one can be used per host at a time. The most commonly used connection plugins are the ``paramiko`` SSH, native ssh (just called ``ssh``), and ``local`` connection types.  All of these can be used in playbooks and with ``/usr/bin/ansible`` to connect to remote machines.
-
-Ansible version 2.1 introduced the ``smart`` connection plugin. The ``smart`` connection type allows Ansible to automatically select either the ``paramiko`` or ``openssh`` connection plugin based on system capabilities, or the ``ssh`` connection plugin if OpenSSH supports ControlPersist.
+Connection plugins allow Ansible to connect to the target hosts so it can execute tasks on them. Ansible ships with many connection plugins, but only one can be used per host at a time. The most commonly used connection plugins are native ``ssh``, ``paramiko``, and ``local``. All of these can be used with the ``ansible`` executable and in playbooks.
```

Comment:
```suggestion
Connection plugins allow Ansible to connect to target hosts so it can execute tasks on them. Ansible ships with many connection plugins, but only one can be used per host at a time. The most commonly used connection plugins are native ``ssh``, ``paramiko``, and ``local``. All of these can be used with ad-hoc tasks and in playbooks.
```